src/Numerical_aux/Transformation.cpp

```diff
@@ -109,135 +109,127 @@ void seissol::transformations::tetrahedronGlobalToReferenceJacobian( real const
 void seissol::transformations::inverseTensor1RotationMatrix( VrtxCoords const i_normal,
                                                              VrtxCoords const i_tangent1,
                                                              VrtxCoords const i_tangent2,
-                                                             DenseMatrixView<3, 3> o_Tinv )
+                                                             yateto::DenseTensorView<2,real,unsigned>& o_Tinv,
+                                                             unsigned row,
+                                                             unsigned col )
 {
```

Comment:
move to line before, doesn't change commit

Reply:
??